dev/how-to/maintain/identify-slow-queries.md

```diff
@@ -1,256 +1,142 @@
 ---
-title: 慢查询日志
+title: 如何找到查询慢的原因
 category: how-to
-aliases: ['/docs-cn/sql/slow-query/']
+aliases: ['/docs-cn/sql/identify-slow-queries/']
```

Comment:
This new alias is wrong and the previous one is correct. ^_^

Reply:
Un `dev/how-to/maintain/slow-log.md`, the alias is `aliases: ['/docs-cn/sql/slow-query/']`, should we use different alias for different documents?

Reply:
@zz-jason `aliases: ['/docs-cn/sql/slow-query/']` is correct. Now it's unnecessary to add `aliases` to the `dev` documents because we have multiple versions and hope to guide the users to v3.0. Only keep `aliases` in v3.0 documents.